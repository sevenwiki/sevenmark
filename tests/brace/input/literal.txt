{{{ Basic literal content with **bold** and *italic* text }}}