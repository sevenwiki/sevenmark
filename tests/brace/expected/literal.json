[
  {
    "LiteralElement": {
      "location": {
        "start": 0,
        "end": 61
      },
      "content": [
        {
          "Text": {
            "location": {
              "start": 4,
              "end": 58
            },
            "content": "Basic literal content with **bold** and *italic* text "
          }
        }
      ]
    }
  }
]